homeassistant/components/switchbee/__init__.py

```diff
@@ -27,13 +28,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     api = CentralUnitAPI(central_unit, user, password, websession)
     try:
         await api.connect()
-    except SwitchBeeError as exp:
-        raise PlatformNotReady("Failed to connect to the Central Unit") from exp
+    except (SwitchBeeError, ClientConnectorError) as exp:
```

Comment:
It's better that the library catches expected aiohttp errors and raises library specific exceptions, as the library is already handling some aiohttp response errors.

Reply:
Done in `pyswitchbee==1.4.8`